doc/tutorial/02_layout/index.md

```diff
@@ -157,6 +163,15 @@ Therefore, we recommend not deleting the files including the built indexes.
 To create an index and thus a layout, the individual samples of the data set are chopped up into k-mers and determine in
 their so-called bin the specific bit setting of the Bloom Filter by passing them through hash functions. This means that
 a k-mer from sample `i` marks in bin `i` with `j` hash functions `j` bits with a `1`.
+
+Example:
+Query ACGT with kmers ACG, CGT.
+Sample 1: AATGT, Sample 2: ACCGT, Sample 3: ACGTA
+2 Hash funktions
+-> Bins 1 to 3 for ACG could look like |0000|0000|0101|
+   Bins 1 to 3 for CGT could look like |0000|0110|1100|
+-> The query seems to match Sample 3
+
 If a query is then searched, its k-mers are thrown into the hash functions and looked at in which bins it only points
 to ones. This can also result in false positives. Thus, the result only indicates that the query is probably part of a
 sample.
```

Comment:
Down below when mentioning the `https://hur.st/bloomfilter/` website, we should mention that the number of inserted elements is the number of kmers in a single bin, and you want to use the biggest bin to be sure.

Thinking about it, does it fit here? Because the layout kinda does all the computations for you?
It does fit, if we rephrase it a bit like: You can look for optimal parameter settings, because we don't optimize the number of hash functions

Reply:
I copied this paragraph from the index tutorial because I thought you might need it here too. But I can also take it out completely. -> https://github.com/seqan/raptor/blob/main/doc/tutorial/03_index/index.md#general-idea--main-parameters